CHANGELOG.md

```diff
@@ -6,9 +6,9 @@
 
 * This library now requires a minimum deployment target of iOS 12.0 or above. iOS 11._x_ is no longer supported. ([#4142](https://github.com/mapbox/mapbox-navigation-ios/pull/4142))
 * MapboxCoreNavigation now requires [MapboxDirections v2.8.0-alpha.2](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v2.8.0-alpha.2). ([#4183](https://github.com/mapbox/mapbox-navigation-ios/pull/4183))
-* MapboxCoreNavigation now requires [MapboxNavigationNative v116._x_](https://github.com/mapbox/mapbox-navigation-native-ios/releases/tag/116.0.0). ([#4183](https://github.com/mapbox/mapbox-navigation-ios/pull/4183))
+* MapboxCoreNavigation now requires [MapboxNavigationNative v117._x_](https://github.com/mapbox/mapbox-navigation-native-ios/releases/tag/117.0.0). ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))
 * MapboxNavigation now requires [MapboxSpeech v2._x_](https://github.com/mapbox/mapbox-speech-swift/releases/tag/v2.1.0). ([#4142](https://github.com/mapbox/mapbox-navigation-ios/pull/4142))
-* MapboxNavigation now requires [MapboxMaps v10.9.0-beta.2](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.9.0-beta.2). ([#4183](https://github.com/mapbox/mapbox-navigation-ios/pull/4183))
+* MapboxNavigation now requires [MapboxMaps v10.9.0-rc.1](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.9.0-rc.1). ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))
 
 ### Routing
```

Comment:
Suggested changelog entries:

### Location tracking

* Fixed an issue where the user’s location history was sometimes matched to unnavigable paths, leading to unreliable location snapping. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))
* Fixed an issue where the `RoadObjectMatcher.matchOpenLR(location:identifier:)` method would incorrectly match some TPEG OpenLR identifiers. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))

### Routing

* Fixed an issue where some alternative routes did not begin at the user’s current location or did not end at the same location as the main route. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))
* Fixed an issue where some continuous alternative routes were unavailable or not consistently available when the user is traveling at low speed. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))
* Fixed an issue where the values of `RouteLeg.segmentDistances` and `RouteLeg.expectedTravelTime` did not add up to `RouteLeg.distance` and `RouteLeg.expectedTravelTime`, respectively. Each value of `RouteLeg.expectedTravelTime` that immediately follows an intersection now includes the time required to traverse the intersection. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))
* Fixed an issue where some alternative routes were identical to the main route due to suppressing a [rat run](https://en.wikipedia.org/wiki/Rat_running). ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))
* Fixed an issue where routes avoided roads with the deprecated tags [`hov=lane`](https://taginfo.openstreetmap.org/tags/hov=lane) and `hov:conditional=lane @ …` as restricted roads. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))

### Guidance instructions

* Turn lane indications now appear even if they slightly differ from what the routing engine would have inferred from the road geometry, For example, turn lanes now appear where the user is expected to use a lane marked as as a right turn lane to make a slight right turn. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))
* Route shields now respect language-specific route numbers in Japan. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))
* Improved the timing of some spoken instructions. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))

Reply:
* Fixed an issue where the current road name label appeared even while the user is not traveling on a known road. ([#4191](https://github.com/mapbox/mapbox-navigation-ios/pull/4191))